specification/azuredatatransfer/AzureDataTransfer.Management/models.tsp

```diff
@@ -0,0 +1,1398 @@
+ * Azure operation completed successfully.
+ */
+model AzureResourceManagerArmResponseConnection {
+  /**
+   * The body type of the operation request or response.
+   */
+  body: Connection;
+}
+
+/**
+ * Azure operation completed successfully.
+ */
+model AzureResourceManagerArmResponseFlow {
+  /**
+   * The body type of the operation request or response.
+   */
+  body: Flow;
+}
+
+/**
+ * Azure operation completed successfully.
+ */
+model AzureResourceManagerArmResponsePipeline {
+  /**
+   * The body type of the operation request or response.
+   */
+  body: Pipeline;
+}
+
+@@clientName(ListApprovedSchemasRequest, "listApprovedSchemasRequest");
+@@clientName(ValidateSchemaResult, "validateSchemaResult");
+@@clientName(SelectedResource, "selectedResource");
+@@clientName(FlowProperties, "flowProperties");
+@@clientName(Flow, "flow");
+@@clientName(FlowsPatch, "flowsPatch");
+@@clientName(ConnectionProperties, "connectionProperties");
+@@clientName(Connection, "connection");
+@@clientName(ConnectionsPatch, "connectionsPatch");
+@@clientName(PipelineProperties, "pipelineProperties");
+@@clientName(Pipeline, "pipeline");
+@@clientName(PipelinesPatch, "pipelinesPatch");
+@@clientName(SchemasListResult, "schemasListResult");
+@@clientName(Schema, "schema");
+@@clientName(FlowType, "flowType");
+@@clientName(ConnectionIdList, "connectionIdList");
+@@clientName(ResourceBody, "resourceBody");
+@@clientName(MessagingOptions, "messagingOptions");
+@@clientName(ApiFlowOptions, "apiFlowOptions");
+@@clientName(StreamSourceAddresses, "streamSourceAddresses");
+@@clientName(SetStreamPassphrase, "setStreamPassphrase");
+@@clientName(SetSourceAddresses, "setSourceAddresses");
+@@clientName(SetDestinationEndpoints, "setDestinationEndpoints");
+@@clientName(SetDestinationEndpointPorts, "setDestinationEndpointPorts");
+@@clientName(Action, "action");
+@@clientName(OperationStatusEnum, "operationStatusEnum");
+@@clientName(Direction, "direction");
+@@clientName(ActionType, "actionType");
+@@clientName(LinkStatus, "linkStatus");
+@@clientName(ProvisioningState, "provisioningState");
+@@clientName(DataType, "dataType");
+@@clientName(StreamProtocol, "streamProtocol");
+@@clientName(ConnectionStatus, "status");
+@@clientName(ForceDisabledStatus, "forceDisabledStatus");
+@@clientName(FlowStatus, "flowStatus");
+@@clientName(TargetType, "targetType");
+@@clientName(FlowBillingTier, "flowBillingTier");
+@@clientName(PipelineStatus, "pipelineStatus");
+@@clientName(SchemaStatus, "schemaStatus");
+@@clientName(ValidateSchemaStatus, "validateSchemaStatus");
+@@clientName(ListFlowsByPipelineConnection, "listFlowsByPipelineConnection");
+@@clientName(InternalMetadataProperties, "internalMetadataProperties");
+@@clientName(OperationStatusProperties, "operationStatusProperties");
+@@clientName(PendingFlow, "pendingFlow");
+@@clientName(PendingConnection, "pendingConnection");
+@@clientName(Subscriber, "subscriber");
+@@clientName(PipelineConnection, "pipelineConnection");
```

Comment:
I'm puzzled about the naming, were things already named lowercase, counter to usual convention?